src/ape/cli/groups.py

```diff
@@ -0,0 +1,12 @@
+import click
+
+from ape.cli.commands import NetworkBoundCommand
+
+
+def ape_group(*args, **kwargs):
+    return click.group(cls=ApeGroup, *args, **kwargs)
+
+
+class ApeGroup(click.Group):
```

Comment:
Note: some CLI options don't need a network connection e.g. `ape compile`

Reply:
Well, this is just a normal click group. You can still make non-network commands with this group. You just now have the ability to create network commands if needed

Reply:
It is so you can do `cli.network_bound_command()` from within a plugin where `cli` is a `click.Group` (an `ApeGroup` specifically!)